docs/dev/README.md

```diff
@@ -49,39 +49,29 @@ general.
 The project comes with a requirements file you can pass to `pip` to install all
 the dependencies needed to work with any check. From the root of the repo, run:
 
-```
-pip install -r requirements-dev.txt
-```
+  pip install -r requirements-dev.txt
```

Comment:
Why remove the formatting for all of the command lines ?

Reply:
🤦‍♂️ that's me using the markdown linter, fixing...